lib/cn/wbt_reader.c

```diff
@@ -88,29 +87,23 @@ bool
 wbti_seek(struct wbti *self, struct kvs_ktuple *seek)
 {
     switch (self->wbd->wbd_version) {
-        case WBT_TREE_VERSION6:
-        case WBT_TREE_VERSION5:
-            return wbti5_seek(self, seek);
-        case WBT_TREE_VERSION4:
-        case WBT_TREE_VERSION3:
-            return wbti4_seek(self, seek);
+        case WBT_TREE_VERSION:
+            return wbti6_seek(self, seek);
     }
 
+    assert(self->wbd->wbd_version == WBT_TREE_VERSION);
     return false;
 }
```

Comment:
We must have previously validated that version=6 right?  So no need to check here.  Can't we just call wbti6_seek directly?  Or just inline the code and remove the specialized 6.c file?  If the next version requires an alternate seek, then we'd have to put this back (or something like it), but there's no telling what v7 will bring along.  It may not impact the seek method.  I think the same applies to wbti6_next, wbti6_reset and wbtr6_read_vref.

Reply:
Sure, will remove this assert.

I left the above switch for future OMF changes but I see your point. Will pull the functionality from wbt_reader_v6.c into wbt_reader.c and remove wbt_reader_v6.c file.